crates/foundry/evm/core/src/backend/mod.rs

```diff
@@ -1076,12 +1111,15 @@ impl DatabaseExt for Backend {
 
     fn create_fork(&mut self, create_fork: CreateFork) -> eyre::Result<LocalForkId> {
         trace!("create fork");
-        let (fork_id, fork, _) = self.forks.create_fork(create_fork)?;
+        let (fork_id, fork, _, fork_block_number) = self.forks.create_fork(create_fork)?;
```

Comment:
It seems like `fork_block_number` can be extracted from `CreateFork`

Reply:
Fixed in https://github.com/NomicFoundation/edr/commit/74a47d17aa564b3169657b43397285f2ad62c31e